core/domain/activity_services.py

```diff
@@ -25,6 +25,15 @@
 
 
 def get_featured_activity_references():
+    """
```

Comment:
Please follow format here: https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments

In particular, the docstring should start immediately after the triple quotes. It does not have to start on a new line.